### PR TITLE
Converting FAQ to new Admin UI

### PR DESCRIPTION
**Pull request by weerd**
### What's this PR do?

This pull request converts the Admin FAQ page to the new Admin UI:

https://user-images.githubusercontent.com/105849/129636713-546cc6e9-94e3-4877-aa20-b90bd15595fe.mp4

### How should this be reviewed?

👀 

### Any background context you want to provide?

I wanted to try out setting up a controller-less rendered page via Inertia.

### Relevant tickets

References [Pivotal #179016026](https://www.pivotaltracker.com/story/show/179016026).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/admin/components/ReviewablePost.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ const ReviewablePost = ({ post }) => {
         {post.status !== 'PENDING' ? (
           <div className="container__row">
             <h4>
-              Tags <HelpLink to="/faq#tags" title="Tag definitions" />
+              Tags <HelpLink to="/admin/faq#tags" title="Tag definitions" />
```
Comment (weerd) [inline resources/assets/admin/components/ReviewablePost.js:181]: Typo missed in prior update when merging Rogue into Northstar.

File: resources/js/Shared/components/Card/NoteCard.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const getNoteIconByType = type => {
+  switch (type) {
+    case 'information':
+      return (
+        <InfoIcon className="flex-initial h-6 ml-4 mt-4 text-blurple-500 w-6" />
+      );
+  }
+};
+
+const NoteCard = ({ className, children, type }) => {
```
Comment (mendelB) [inline resources/js/Shared/components/Card/NoteCard.js:16]: This is neat! 🤩

**Review by mendelB — APPROVED** [review]
Love it!! 🎨

File: resources/js/Shared/components/Card/NoteCard.js (hunk 2 above)
Comment (weerd) [reply resources/js/Shared/components/Card/NoteCard.js:16]: Ya! Hoping to expand it so we can have "warning", "error" etc note cards apart from just "information" 😄

**Pull request merged**